modules/experimental-layers/src/advanced-text-layer/advanced-text-layer.js

```diff
@@ -37,14 +37,14 @@ const DEFAULT_COLOR = [0, 0, 0, 255];
 // };
 
 const defaultProps = {
-  getText: x => x.text,
-  getPosition: x => x.coordinates,
-  getColor: x => x.color || DEFAULT_COLOR,
-  getSize: x => x.size || 32,
-  getAngle: x => x.angle || 0,
-  getTextAnchor: x => x.textAnchor || 'middle',
-  getAlignmentBaseline: x => x.alignmentBaseline || 'center',
-  getPixelOffset: x => x.pixelOffset || [0, 0],
+  getText: {type: 'accessor', value: x => x.text},
```

Comment:
Nit: My thinking was that we would just auto-infer `accessor` if the prop name starts with `get` and is function-valued?

Reply:
This is something we removed from 6.2 to 6.3. There was too much magic going on in the auto-infer system and it got very confusing when custom layers do not update as expected.